sdl_android_lib/src/com/smartdevicelink/streaming/AbstractPacketizer.java

```diff
@@ -9,12 +9,12 @@
 abstract public class AbstractPacketizer {
 
 	protected IStreamListener _streamListener = null;
-
+	private final static int BUFF_READ_SIZE = 1000000;

```

Comment:
Should be updated based on [protocol spec](https://github.com/smartdevicelink/protocol_spec/issues/4)